src/smartthings/smartapps/smartthings/amazon-echo-home-skill.src/amazon-echo-home-skill.groovy

```diff
@@ -886,59 +890,85 @@ def deviceHeartbeatCheck() {
+			case "Dimmer Switch":
+			case "Z-Wave Dimmer Switch Generic":
+				def msr = "${device?.getZwaveInfo()?.mfr}-${device?.getZwaveInfo()?.prod}-${device?.getZwaveInfo()?.model}"
+				if (msr != null) {
+					switch (msr) {
+						case "001D-1B03-0334":  // ZWAVE Leviton In-Wall Switch (dimmable) (DZMX1-1LZ)
+						case "001D-1C02-0334":  // ZWAVE Leviton In-Wall Switch (non-dimmable) (DZS15-1LZ)
+						case "001D-1D04-0334":  // ZWAVE Leviton Receptacle (DZR15-1LZ)
+						case "001D-1A02-0334":  // ZWAVE Leviton Plug in Appliance Module (Non-Dimmable) (DZPA1-1LW)
+						case "001D-1902-0334":  // ZWAVE Leviton Plug in Lamp Dimmer Module (DZPD3-1LW)
+						case "0063-4952-3031":  // ZWAVE Jasco In-Wall Smart Outlet (12721)
+						case "0063-4952-3033":  // ZWAVE Jasco In-Wall Smart Switch (Toggle Style) (12727)
+						case "0063-4952-3032":  // ZWAVE Jasco In-Wall Smart Switch (Decora) (12722)
+						case "0063-5052-3031":  // ZWAVE Jasco Plug-in Smart Switch (12719)
+						case "0063-4F50-3031":  // ZWAVE Jasco Plug-in Outdoor Smart Switch (12720)
+						case "0063-4944-3031":  // ZWAVE Jasco In-Wall Smart Dimmer (Decora) (12724)
+						case "0063-4944-3032":  // ZWAVE Jasco In-Wall 1000 Watt Smart Dimmer (Decora) (12725)
+						case "0063-4944-3033":  // ZWAVE Jasco In-Wall Smart Dimmer (Toggle Style) (12729)
+						case "0063-5044-3031":  // ZWAVE Jasco Plug-in Smart Dimmer (12718)
+						case "0063-4944-3034":  // ZWAVE Jasco In-Wall Smart Fan Control (12730)
+							timeout = 60
+							timeoutReason = "device type is $deviceTypeName and msr $msr is in the list of heartbeat supported devices"
+							break
+					}
+				}
+				break
+		}
+
+		// def msr = device.device?.getDataValue("MSR")
+		// Check the Data
+		if (timeout == 0 && (deviceTypeName.startsWith('ZigBee') || deviceTypeName.startsWith('ZLL'))) {
+			String dataManufacturer = device.device?.getDataValue("manufacturer")
+			String dataApplication = device.device?.getDataValue("application")
+			String dataModel = device.device?.getDataValue("model")
+			String dataEval = [dataManufacturer, dataApplication, dataModel].toString()
+			switch (dataEval) {
+				case ['OSRAM', '07', 'PAR16 50 TW'].toString():  // UK
+				case ['OSRAM', '03', 'Flex RGBW'].toString():  // UK
+				case ['OSRAM', '15', 'Classic A60 W clear - LIGHTIFY'].toString():  // UK
+				case ['OSRAM', '14', 'Classic B40 TW - LIGHTIFY'].toString():  // UK
+					timeout = 35
+					timeoutReason = "device type is a ZigBee device and product data is recognized ($dataEval)"
+					break
+			}
+		}
+
+		// Check DTHs with ambiguous names in type name
+		if (timeout == 0) { // Still!
+			switch (device.name) {
+				case "OSRAM LIGHTIFY LED Tunable White 60W": // US
+				case "OSRAM LIGHTIFY LED FLEXIBLE STRIP RGBW": // UK - belt and suspenders
+				case "OSRAM LIGHTIFY LED PAR16 50 Tunable White GU10": // UK - belt and suspenders
+				case "OSRAM LIGHTIFY LED Smart Connected Light": // UK - belt and suspenders
```

Comment:
Honestly, no. If a US device meets the same criteria as a UK one, it'll get a heartbeat timeout, similar to how the SmartPower Outlet just worked.